### ISSUE TITLE
Choose an alternative to Swagger for API documentation

### ISSUE THREAD

**Issue by ahamelers** [datadryad/dryad-product-roadmap#4032]
As identified in our VPAT assessment, Swagger's auto-generated documentation tool has a lot of accessibility problems. Many of these appear to be longstanding without a proposed timeline for resolution: https://github.com/swagger-api/swagger-ui/issues/7350

Seek alternatives to Swagger for API doc generation, preferably one that also uses the OpenAPI specification so we have to do minimal YAML edits. It should be embeddable, and generally meet accessibility requirements (checking that output has no alerts or errors with [WAVE](https://wave.webaim.org/) and axe DevTools should be sufficient)

Somewhat alarmingly, there might be a _lot_ of these to choose from: https://openapi.tools/#documentation

**Comment by ahamelers** [comment]
Absolutely none of the tools met accessibility standards. I started contributing to a project to get to something that was!
